war/src/main/less/base/yui-compatibility.less

```diff
@@ -95,6 +95,14 @@
   color: inherit;
 }
 
+.ygtvfocus {
```

Comment:
I'd put a comment in this section explaining what it's for

Reply:
Done. Is this the kind of thing you meant?